<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolcanicRelicParam</class>
 <widget class="QWidget" name="VolcanicRelicParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_top">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_select_players">
     <property name="title">
      <string>选择账号</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_select_player">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_select_1p">
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>房主：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_select_1p">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2P</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_select_2p">
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>房客：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_select_2p">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2P</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_select_level">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>选择关卡</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_select_level" stretch="1,2,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_level">
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>关卡：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_level">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>果仁瀑布</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>榛子瀑布</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>蓝莓丛林</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黑提丛林</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>奶昔基地</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>圣代基地</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>85</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_select_plan">
     <property name="title">
      <string>选择方案</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_select_plan">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_plan_path">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>方案路径：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_plan_path">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_plan_path">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1p_plan">
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>放卡方案：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_1p_plan">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_view_plan">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>查看方案</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_level_setting">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
     <property name="title">
      <string>通关设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_loop_count">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>循环次数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_loop_count">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_flop_pos">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>翻牌位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_flop_pos">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>1;2;3;4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>135</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
